addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -120,6 +120,25 @@ test("create sub thread from existing message (slow network)", async () => {
     });
 });
 
+test("Shows 'New Thread' on sub-thread name from empty message", async () => {
```

Comment:
This is only testing the mock server. We should add a test in `test_discuss_sub_channels` instead.